litellm/model_prices_and_context_window_backup.json

```diff
@@ -2004,9 +2004,9 @@
         "cache_read_input_token_cost": 1.25e-07,
         "input_cost_per_token": 1.25e-06,
         "litellm_provider": "azure",
-        "max_input_tokens": 272000,
-        "max_output_tokens": 128000,
-        "max_tokens": 128000,
+        "max_input_tokens": 128000,
+        "max_output_tokens": 16384,
```

Comment:
we follow the pattern of setting max_tokens== max_output_tokens in the codebase 

max_tokens == context window